### ISSUE TITLE
News nicht übersetzt

### ISSUE THREAD

**Issue by AurelioSilver** [AurelioSilver/pillarsofeternity-2-german-patch#39]
Anscheinend lädt er die anhand der Sprach ID nach.und de_patch kennt er nicht.

Eventuell geht das mit dem Override Ordner, allerdings müßte man dann alle Texte aus den Erweiterungen in eine Datei bringen...

**Comment by Xaratas** [comment]
Die gibts in übersetzter form?

**Comment by AurelioSilver** [comment]
Ja, wenn ich beim aktuellen Patch auf DE stelle, sind die übersetzt... Die Texte werden wohl online gezogen, wie es scheint.

**Comment by Xaratas** [comment]
Ja, da gibts einen rss feed für. Aber den hatte ich noch nie übersetzt gesehen (gut lange nicht mehr betrachtet)
